stub-generator/matrix_analytics_stub_generator/kotlin.py

```diff
@@ -26,11 +26,15 @@ def compute_kotlin(schema: Schema) -> str:
 
     if is_screen:
         itf = "VectorAnalyticsScreen"
-    else:
+    elif schema.event_name:
```

Comment:
this is my bad, forgot to mention in the other comment, we'll also need to drop the `override` when the itf is blank https://github.com/matrix-org/matrix-analytics-events/pull/28/files#diff-7334dd3531795ac0894703f568db3f5afd7f0410dd5a61abf44f2a78171510a7R60

https://github.com/matrix-org/matrix-analytics-events/pull/28/files#diff-6d7172fb23e30fc3d2fd5fe0c9d311f76085b87e71a19ec37e3f2eba534897a7R100

https://github.com/matrix-org/matrix-analytics-events/pull/28/files#diff-6d7172fb23e30fc3d2fd5fe0c9d311f76085b87e71a19ec37e3f2eba534897a7R102